orchestrator/db/database.py

```diff
@@ -187,7 +186,9 @@ class Database:
     def __init__(self, db_url: str) -> None:
         self.request_context: ContextVar[str] = ContextVar("request_context", default="")
         self.engine = create_engine(db_url, **ENGINE_ARGUMENTS)
-        self.session_factory = sessionmaker(bind=self.engine, **SESSION_ARGUMENTS)
+        self.session_factory = sessionmaker(
+            bind=self.engine, class_=WrappedSession, autocommit=False, autoflush=True, query_cls=SearchQuery
```

Comment:
Yes, mypy complains because it is now enforced that these are keyword arguments. The solution that still uses the SESSION_ARGUMENTS looks a bit bulky. I prefer the direct approach so I can see what the values are.

Reply:
Yeah, but somehow the tests should use the same configuration as the real application. Maybe a separate function that calls `sessionmaker()`? Then the test code can call that and only override the `bind` param

Reply:
I'll create a cleanup/followup ticket that addresses the issues mentioned in this ticket, as well as the remaining SqlAlchemy 2.0 deprecations (that will be removed in 3.0)